packages/SwingSet/src/kernel/virtualObjectManager.js

```diff
@@ -364,8 +364,9 @@ export function makeVirtualObjectManager(
           // exported non-virtual object: Remotable
           const remotable = requiredValForSlot(vref);
           if (remotableRefCounts.has(remotable)) {
-            /** @type {number} */
-            const oldRefCount = (remotableRefCounts.get(remotable));
+            const oldRefCount = /** @type {number} */ (remotableRefCounts.get(
```

Comment:
I'll second @turadg 's avoidance of inline types here, also Chip has a PR in the works that touches this file, so if we can avoid changing it, that will probably land more smoothly.

Reply:
It won't pass linting if I don't update this file. Happy to switch to define the const as `any` instead of casting, but I actually prefer the conflict generation in this case as it'd be easy to forget an `any` declaration.

Reply:
Inline casts will summon Angry Dean Face, and nobody wants that.

Reply:
Then I need a behavior change, maybe the one suggested by @turadg to default to `0`.

Reply:
Rewrote both occurrences of this case. One to transform `.has` + `.get` into a `.get` and `=== undefined` check, the other to add an `!== undefined` assertion.
@FUDCo PTAL